_health-care/_js/components/personal-information/NameAndGeneralInfoSection.jsx

```diff
@@ -32,8 +32,11 @@ class NameAndGeneralInfoSection extends React.Component {
             <div className="small-12 columns">
               <h4>Date of Birth</h4>
               <span className="usa-form-hint usa-datefield-hint" id="dobHint">For example: 04 28 1986</span>
-              <DateInput date={this.props.data.dateOfBirth}
-                  onUserInput={(update) => {this.props.onStateChange('dateOfBirth', update);}}/>
+              <DateInput
+                  day={this.props.data.dateOfBirth.day}
```

Comment:
We're getting console errors because `applicationData` has strings for the date values instead of numbers (and `propTypes` specifies it should be a number value).